src/locales/en.ts

```diff
@@ -43,6 +53,10 @@ const en: AppTranslations = {
             buttons: {
                 signInWithFacebook: 'Sign in with Facebook',
                 signInWithCredentials: 'Sign in'
+            },
+            errors:{
+                signInWithFacebook: "Error occurred while login whith facebook. Incorrect email or password.",
```

Comment:
'An error occurred while **logging in with** facebook' é suficiente aqui.
Coloca a tradução equivalente no pt.ts.